scipy/sparse/linalg/_propack/meson.build

```diff
@@ -84,11 +84,6 @@ elements = [
 ]
 
 fargs_propack = ['-U_OPENMP']
-if use_g77_abi
-  # FIXME: define needs to be removed from PROPACK code, in favor of using
-  # `wcdotc` unconditionally, like is done in ARPACK.
-  fargs_propack += ['-DSCIPY_USE_G77_CDOTC_WRAP=1']
-endif
```

Comment:
Can we split off all G77-related changes into a separate PR?

Reply:
Great idea! Now that the G77 wrappers are separate from `_generate_blas_wrappers.py`, this is finally possible #19855.